marketing_emails/email_kickoff_no_keynote.md

```diff
@@ -0,0 +1,30 @@
+*We are back!*
+
+PyTN is returning February 10th and 11th 2018.  Bill Israel, PyTN Chairman, and his team are extending Jason Myers' success of previous years.
```

Comment:
I'd push for these emails to be largely the same, except for the keynoters section.

Maybe combine some elements of both this and the one above? I like bits of both of them. I like the information about the event in the one with keynoters, and I like the "steps" in the second one with the mention of the twitter info.